generators/server/templates/src/test/java/package/web/rest/AccountResourceIT.java.ejs

```diff
@@ -108,8 +108,7 @@ import static org.springframework.security.test.web.servlet.response.SecurityMoc
 <%_ } else { _%>
 @AutoConfigureMockMvc
 <%_ } _%>
-@WithMockUser(value = TEST_USER_LOGIN)
-@IntegrationTest
```

Comment:
This one needs the @IntegrationTest annotation since it has methods using a DB, thats whats causing the test failures. @dinu0000 can you try adding it back for this template